src/amo/components/SearchResult/index.js

```diff
@@ -35,6 +36,7 @@ type InternalProps = {|
 
 export class SearchResultBase extends React.Component<InternalProps> {
   static defaultProps = {
+    isAllowedOrigin: defaultIsAllowedOrigin,
```

Comment:
so I set this up this way for testing (this was how I interpreted part of your comment [here](https://github.com/mozilla/addons-frontend/pull/5225#discussion_r203726631)

Reply:
It is a good pattern! I think we tend to name the prop with a leading underscore (`_isAllowedOrigin`) so that there is no need to alias the import + we know that this prop is injected with a default value/implementation in the code (because of the underscore).